simpleflow/activity.py

```diff
@@ -37,7 +37,7 @@ def wrap(func):
     return wrap
 
 
-class Activity(object):
+class ActivityType(object):
```

Comment:
Ahah ;)

Re-reading SWF docs now, I'm not even sure. http://docs.aws.amazon.com/amazonswf/latest/developerguide/swf-dev-tasks.html

We needed to differentiate the definition of the function from the object wrapping an execution instance. But it seems it's already Activity VS ActivityTask, which matches what's currently done in simpleflow.